style.css

```diff
@@ -497,4 +494,79 @@ hr {
     position: absolute;
     margin-top: 200px;
   }
+
+  .contact {
+    text-align: center;
+  }
+
+  .contact h1 {
+    color: #9390ff;
+    align-items: center;
+    text-align: center;
+    font-family: roboto, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    margin-top: 120px;
+    margin-bottom: 24px;
+  }
+
+  .contact p {
+    width: 327px;
+    color: #fff;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    margin: 0 24px;
+    margin-bottom: 40px;
+  }
+
+  .form-container {
+    display: flex;
+    flex-direction: column;
+  }
+
+  #name,
+  #mail {
+    margin: 0 24px;
+    padding: 15px 16px;
+    width: 290px;
+    height: 18px;
+    margin-bottom: 24px;
+  }
+
+  #message {
+    padding: 12px;
+    margin: 0 24px;
+    width: 300px;
+    height: 148px;
+    margin-bottom: 40px;
+  }
+
+  footer {
+    text-align: center;
+  }
+
+  .form-button {
+    padding: 12px;
+    font-family: roboto, sans-serif;
+    font-weight: 500;
+    font-size: 12px;
+    background: #7d73fc;
+    color: #edecff;
+    align-self: center;
+    width: 89px;
+    height: 48px;
+    border: none;
+    margin-bottom: 71px;
+  }
+
+  .footer-email {
+    font-family: roboto, sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    color: #ebebff;
+    margin-top: 0;
+  }
 }
```

Comment:
- [ ] Kindly fix the layout (colours, font, size, space, margins) according to the Figma design. You can open the browser as chrome from vscode and fix it.
- [ ] Kindly add button interactions such as hover, active when pressed according to the Figma design.
- Your design:
<img width="367" alt="Screenshot 2022-03-22 at 11 58 38 PM" src="https://user-images.githubusercontent.com/70269507/159575826-285cccf5-8164-42d2-8b47-d5559f006ff0.png">

- Figma design:
<img width="343" alt="Screenshot 2022-03-23 at 12 05 47 AM" src="https://user-images.githubusercontent.com/70269507/159575946-313154e8-d013-46c2-bc74-37750ff1b0c7.png">